src/Filesystem/Asset.php

```diff
@@ -38,6 +41,31 @@ public function __construct(string $path)
 		]);
 	}
 
+	/**
+	 * Magic caller for asset methods
+	 *
+	 * @throws \Kirby\Exception\BadMethodCallException
+	 */
+	public function __call(string $method, array $arguments = [])
+	{
+		// public property access
+		if (isset($this->$method) === true) {
+			return $this->$method;
+		}
+
+		// asset methods
+		if ($this->hasMethod($method)) {
+			return $this->callMethod($method, $arguments);
+		}
+
+		// asset method proxy
+		if (method_exists($this->asset(), $method)) {
+			return $this->asset()->$method(...$arguments);
+		}
```

Comment:
Tiny issue: The order of these two cases should be swapped for consistency with `File::__call()`.